content/md/permissions-settings/manage-the-web-api-permissions_fr.md

```diff
@@ -0,0 +1,91 @@
+---
+id: manage-the-web-api-permissions
+themes: permissions-settings
+title: Gérer vos permissions des **API Web**
+popular: false
+ee-only: false
+related: build-your-user-roles, what-is-a-role, what-is-a-user, manage-the-interface-and-actions-accesses
+---
+
+# Définir les autorisations sur l'API Web
+
+Les autorisations sur l'[API Web](https://api.akeneo.com) sont accordées en fonction du(des) [rôle(s) de l'utilisateur](what-is-a-role.html).
+
+Pour modifier les autorisations de l'API Web pour un rôle d'utilisateur :
+1. Allez sur `Système` et cliquez sur l'entrée du menu `Rôles`
+1. Cliquez sur le rôle à modifier
+1. Cliquez sur l'onglet `Autorisations de l'API Web`
+    ![image](System_Roles_WebAPIPermissions_fr.png)
```

Comment:
This image is too wide (more than 840px).

Reply:
Resize done